safe-logging/src/main/java/com/palantir/logsafe/SafeLoggable.java

```diff
@@ -22,9 +22,9 @@
 public interface SafeLoggable {
 
     /** The format of the message. Arguments placeholders should be represented by "{}". */
```

Comment:
we're also presently debating whether one should use placeholders, I think we'll end up indicating that placeholders are unnecessary.